lib/rules/declaration-property-value-no-unknown/index.mjs

```diff
@@ -24,6 +24,8 @@ const messages = ruleMessages(ruleName, {
 	rejected: (property, value) => `Unexpected unknown value "${value}" for property "${property}"`,
 	rejectedParseError: (property, value) =>
 		`Cannot parse property value "${value}" for property "${property}"`,
+	rejectedValueFunction: (property, valueFunction) =>
```

Comment:
Let's not add a supplementary type of message.
You can reuse `rejected`, it will remain accurate.